tensilelite/rocisa/rocisa/include/helper.hpp

```diff
@@ -21,15 +21,17 @@
  *
  * ************************************************************************ */
 #pragma once
+#include <iostream>
 #include <stdexcept>
 #include <string>
-#include <sys/wait.h>
 #include <unistd.h>
 #include <vector>
 #ifdef __GNUG__
 #include <cxxabi.h>
 #endif
 
+#include <amd_comgr/amd_comgr.h>
+
```

Comment:
Please land this after my change #1951 and then don't land any more non trivial definitions in the header: put them in helper.cpp

Reply:
I'll put this into cpp

Reply:
Since this patch needs more review/iteration and is a functional change, please stack it onto #1951 which is NFC and makes the structural change. 

If this one was ready to go, I would have approved it and adapted mine, but we shouldn't have two independent patches changing the same structure. 

Generally, NFC/structure patches, which have a low risk of regression should precede functional changes that can introduce regressions and further bugs.

Reply:
Now the PR is on top of #1951